proto/status.proto

```diff
@@ -19,6 +19,7 @@ syntax = "proto2";
 package cockroach.proto;
 option go_package = "proto";
 
+import "cockroach/proto/data.proto";
```

Comment:
This is a proto definition file.  Pretty sure you can't do that.  And we don't do that in any of our proto files.
https://developers.google.com/protocol-buffers/docs/proto